crates/sui-metrics-proxy/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "sui-metrics-proxy"
+version = "0.0.1"
+authors = ["Mysten Labs <build@mystenlabs.com>"]
+license = "Apache-2.0"
+publish = false
+edition = "2021"
+
+[dependencies]
+axum = "0.6.2"
+anyhow = { version = "1.0.64", features = ["backtrace"] }
+clap = { version = "3.2.17", features = ["derive"] }
+tokio = { workspace = true, features = ["full"] }
+tracing = "0.1.36"
+tracing-subscriber = "0.3.16"
```

Comment:
you should probably just use the subscriber we have in the `telemetry-subscriber` crate

Reply:
sure, no problem.